qokit/qaoa_objective_maxcut.py

```diff
@@ -35,7 +35,8 @@ def get_qaoa_maxcut_objective(
     G : nx.Graph
         graph on which MaxCut will be solved
     precomputed_cuts : np.array
-        precomputed cuts to compute the QAOA expectation
+        precomputed cuts to compute the QAOA expectation, for maximization problem
+        send the precomputed cuts/energies as negative
```

Comment:
I think this may be wrong, no? Since we flip the sign below and since `get_qaoa_objective` assumes that the optimization problem is minimization (hence the change to `precomputed_costs`), we **do not** have to pass cuts as negative. Instead, cuts are passed directly as was the case originally?

Let me know if I'm being confused.